packages/preview/plotst-0.1.0/example/main.typ

```diff
@@ -0,0 +1,212 @@
+#import "/lib.typ": *
```

Comment:
I'm a bit on the fence with this one. To make this example copy-pasteable it should use the `@preview` import. That makes it difficult to test locally though, of course.

Reply:
I could put both in and comment one with "for local testing" or something. On my repo that is. I removed the file here

Reply:
Up to you then!